arch/arm64/boot/dts/qcom/sm7325-nothing-spacewar.dts

```diff
@@ -0,0 +1,642 @@
+// SPDX-License-Identifier: BSD-3-Clause
```

Comment:
The commit message format doesn't match the subsystem, should be `arm64: dts: qcom: foobar`

Also you need to add some extra lines as commit message, not just subject. I'd write something like this:

```
arm64: dts: qcom: Add Nothing Phone (1)

Add an initial dts for the Nothing Phone (1) based on the SM7325 SoC.
This SoC is, just like QCM6490, mostly compatible with SC7280 dtsi so create a SM7325 SoC based on that.

Working features:
* list the
* working features

And maybe some more text that you want to add, anything interesting you can think of
```